test_harness/src/lib.rs

```diff
@@ -2225,6 +2254,444 @@ mod tests {
                 );
             }
         }
+
+        // Testing analytics statistics cases
+
+        // 1. CHAIN_1 in an unauthenticated /analytics call filters results and correctly returns count/paid
+        {
+            let analytics = get_analytics(
+                &api_client,
+                &leader.sentry_url,
+                &get_analytics_query(Metric::Count, IMPRESSION, vec![ChainId::new(1)]),
```

Comment:
Can you replace these places by using the static values?
E.g. `GANACHE_1337.chain_id` & `GANACHE_1.chain_id`